### PR TITLE
Updating CMake Debugging Docs to have CMake Debugger

### PR DESCRIPTION
**Pull request by sinemakinci1**
Adding CMake Debugger information and refreshing top-level screenshots

### CODE REVIEW

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
Hi Sinem,
I sent you a mail about some ideas for alt-text.

**Review by TylerMSFT — COMMENTED** [review]
You are missing the :::image-end:::

**Comment by ShannonLeavitt** [comment]
@TylerMSFT

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by sinemakinci1** [comment]
@TylerMSFT It should be good to review now, thank you for your help!

**Review by TylerMSFT — APPROVED** [review]
Thanks, Sinem!

**Comment by TylerMSFT** [comment]
#sign-off

File: docs/build/configure-cmake-debugging-sessions.md [hunk 1]
```diff
@@ -16,14 +16,28 @@ Native CMake support is available in Visual Studio 2017 and later. To see the do
...
 You can also start a debug session from Solution Explorer. First, switch to **CMake Targets View** in the **Solution Explorer** window.
 
-![Screenshot of the CMake targets view command.](media/cmake-targets-view.png  "CMake Targets View menu item")
+:::image type="complex" source="media/switch-to-targets-view.png" alt-text="Screenshot of the CMake Targets View menu":::
+The solution explorer is shown. A right-click on an item in the Folder View has opened a menu. On that menu, the Switch to Targets View item is highlighted.
+:::image-end:::
 
 Then, right-click on an executable and select **Debug**. This command automatically starts debugging the selected target based on your active configuration.
 
+:::image type="complex" source="media/debug-targets-view.png" alt-text="Screenshot of the CMake Targets View debug option menu":::
```
Comment (Court72) [inline docs/build/configure-cmake-debugging-sessions.md:31]:
```suggestion
:::image type="complex" source="media/debug-targets-view.png" alt-text="Screenshot of the CMake Targets View debug option menu.":::
```

File: docs/build/configure-cmake-debugging-sessions.md [hunk 2]
```diff
@@ -16,14 +16,28 @@ Native CMake support is available in Visual Studio 2017 and later. To see the do
...
+When you right click on a target in the CMake Targets view, there is an option to debug a chosen target. The Debug option is highlighted.
+:::image-end:::
+
+Starting in Visual Studio 2022 Version 17.6, you can also start a debugging session on your CMakeLists.txt file. To do so, just set a breakpoint in your CMakeLists.txt file and run **Configure Project with CMake Debugger** from the **Project** dropdown
```
Comment (Court72) [inline docs/build/configure-cmake-debugging-sessions.md:35]:
```suggestion
Starting in Visual Studio 2022 Version 17.6, you can also start a debugging session on your CMakeLists.txt file. To do so, just set a breakpoint in your CMakeLists.txt file and run **Configure Project with CMake Debugger** from the **Project** dropdown.
```

File: docs/build/configure-cmake-debugging-sessions.md [hunk 3]
```diff
@@ -16,14 +16,28 @@ Native CMake support is available in Visual Studio 2017 and later. To see the do
...
+
+:::image type="complex" source="media/cmake-debugger-entry.png" alt-text="Screenshot of the CMake Debugger dropdown":::
```
Comment (Court72) [inline docs/build/configure-cmake-debugging-sessions.md:37]:
```suggestion
:::image type="complex" source="media/cmake-debugger-entry.png" alt-text="Screenshot of the CMake Debugger dropdown.":::
```

**Comment by Court72** [comment]
Merging now.

**Pull request merged**